### PR TITLE
feat(auth): create login, register and logout

### PR DESCRIPTION
**Pull request by Hasban-Fardani**
Feat: #1

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+use App\Http\Controllers\auth\LoginController;
+use App\Http\Controllers\auth\LogoutController;
+use App\Http\Controllers\auth\RegisterController;
+use Illuminate\Support\Facades\Route;
+
+Route::post('/login', LoginController::class);
```
Comment (billalxcode) [inline routes/api.php]: Please add guest middleware in route api

File: routes/api.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Route::post('/logout', LogoutController::class);
```
Comment (billalxcode) [inline routes/api.php]: Please add guest middleware in route api

File: routes/api.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+Route::post('/register', RegisterController::class);
```
Comment (billalxcode) [inline routes/api.php:12]: Please add guest middleware in route api

**Review by billalxcode — CHANGES_REQUESTED** [review]
Please add guest middleware in route api

File: routes/api.php (hunk 1 above)
Comment (Hasban-Fardani) [reply routes/api.php]: okay

**Review by billalxcode — APPROVED** [review]
TYSM!!

**Comment by Hasban-Fardani** [comment]
NOTE: if you get error about package, please run: `composer install`

**Pull request merged**